api/BroadcastChannel.json

```diff
@@ -32,6 +32,9 @@
           "ie": {
             "version_added": false
           },
+          "nodejs": {
+            "version_added": "15.4.0",
```

Comment:
I suggest you add a note that this needs to be imported from "worker_threads" and that it is experimental. That would align with how api.MessageChannel was documented prior to being available as a global.